packages/analytics/dashboard-renderer/src/components/DashboardTile.vue

```diff
@@ -31,8 +31,10 @@ const props = withDefaults(defineProps<{
   context: DashboardRendererContextInternal,
   height?: number,
   queryReady: boolean,
+  refreshCounter?: number,
```

Comment:
Does this need to be optional?  The renderer is the only thing that renders tiles, and it always provides the counter...